docs/reference/commandline/run.md

```diff
@@ -315,31 +337,26 @@ docker run -v c:\foo:c: ...
 docker run -v c:\foo:c:\existing-directory-with-contents ...
 ```
 
-For in-depth information about volumes, refer to [manage data in containers](https://docs.docker.com/engine/tutorials/dockervolumes/)
+For in-depth information about volumes, refer to [the Storage overview](https://docs.docker.com/storage/)
 
+#### Start the Docker daemon using a static binary in the container
 
-### Add bind mounts or volumes using the --mount flag
-
-The `--mount` flag allows you to mount volumes, host-directories and `tmpfs`
-mounts in a container.
-
-The `--mount` flag supports most options that are supported by the `-v` or the
-`--volume` flag, but uses a different syntax. For in-depth information on the
-`--mount` flag, and a comparison between `--volume` and `--mount`, refer to
-the [service create command reference](service_create.md#add-bind-mounts-or-volumes).
-
-Even though there is no plan to deprecate `--volume`, usage of `--mount` is recommended.
-
-Examples:
+This technique is useful in CI/CD set-ups where you need to test using a static
+Docker binary. This allows the container to create and manipulate the daemon on
+its host. This assumes that `/path/to/static-docker-binary` does not exist on
```

Comment:
This won't work; if `/usr/bin/docker` exists in the container, but `/path/to/static-docker-binary` doesn't (in the `-v` case), you'll end up with docker attempting to bind-mount a directory (`/path/to/static-docker-binary`) on top of a file/binary (`/usr/bin/docker`) inside the container.